velox/exec/ContainerRowSerde.cpp

```diff
@@ -292,6 +292,9 @@ void deserializeSwitch(
     ByteStream& in,
     vector_size_t index,
     BaseVector& result) {
+  // This is called only on non-null values, so clear the null flag in
```

Comment:
Is this a fix for a pre-existing bug? If so, we do need to consider memory re-use holistically and come up with a clean and safe pattern(s) to use everywhere. CC: @laithsakka

Reply:
@oerling I wonder if this question got lost. Repeating.

----
Is this a fix for a pre-existing bug? If so, we do need to consider memory re-use holistically and come up with a clean and safe pattern(s) to use everywhere.